[2024-10-01T18:54:36.561+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-01T18:54:36.579+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-01T18:54:24.780871+00:00 [queued]>
[2024-10-01T18:54:36.586+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-01T18:54:24.780871+00:00 [queued]>
[2024-10-01T18:54:36.587+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-10-01T18:54:36.602+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-10-01 18:54:24.780871+00:00
[2024-10-01T18:54:36.611+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_with_cleaning_transfer', 'create_dataset', 'manual__2024-10-01T18:54:24.780871+00:00', '--job-id', '3096', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpjjw4zfwa']
[2024-10-01T18:54:36.612+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=38179) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-01T18:54:36.612+0000] {standard_task_runner.py:91} INFO - Job 3096: Subtask create_dataset
[2024-10-01T18:54:36.612+0000] {standard_task_runner.py:63} INFO - Started process 38181 to run task
[2024-10-01T18:54:36.648+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_with_cleaning_transfer.create_dataset manual__2024-10-01T18:54:24.780871+00:00 [running]> on host 71321b93f95e
[2024-10-01T18:54:36.721+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_with_cleaning_transfer' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T18:54:24.780871+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T18:54:24.780871+00:00'
[2024-10-01T18:54:36.722+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-01T18:54:36.730+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-10-01T18:54:36.735+0000] {base.py:84} INFO - Using connection ID 'my-gcp-conn' for task execution.
[2024-10-01T18:54:36.789+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value finalproject_data.
[2024-10-01T18:54:36.789+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value madt-finalproject.
[2024-10-01T18:54:36.790+0000] {bigquery.py:481} INFO - Creating dataset: finalproject_data in project: madt-finalproject 
[2024-10-01T18:54:37.118+0000] {bigquery.py:2045} INFO - Dataset finalproject_data already exists.
[2024-10-01T18:54:37.132+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-01T18:54:37.139+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl_with_cleaning_transfer, task_id=create_dataset, execution_date=20241001T185424, start_date=20241001T185436, end_date=20241001T185437
[2024-10-01T18:54:37.187+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-01T18:54:37.208+0000] {taskinstance.py:3482} INFO - 8 downstream tasks scheduled from follow-on schedule check
[2024-10-01T18:54:37.216+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
